nbox/utils.py

```diff
@@ -2,13 +2,19 @@
 
 import os
 import io
+import logging
 from PIL import Image
 
+logging.basicConfig(level="INFO")
+def info(x: str, *args):
+  x = str(x)
+  for y in args:
+    x += " " + str(y)
+  logging.info(x)
```

Comment:
Doing `str(y) for y in args` may not always be the best (some objects might want to be `repr`ed), just pass args to `info()` of logging.

Reply:
Will replace it with
```python
def info(x: str, *args):
  # because logging.info requires formatted strings
  x = repr(x)
  x = " ".join([x] + [repr(y) for y in args])
  logging.info(x)
```